product_unavailable/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import product, website_sale
```

Comment:
from . import product_unavailable

However, the file is named wrong and should be `product_product` instead, to match the model name.